test/cppsuite/test_harness/util/debug_utils.h

```diff
@@ -68,7 +68,7 @@ static bool _include_date = true;
 /* Mutex used by logger to synchronize printing. */
 extern std::mutex _logger_mtx;
```

Comment:
Do we want a `debug_utils.cxx`? We'd be able to put the mutex definition there instead of `run.cxx`.

Reply:
That would be great

Reply:
There's no class in the .h file. We need to introduce class for logger.

Reply:
That's a separate concern I think. You can still have a `cxx` file that contains the function definitions and the mutex.

Reply:
I've added debug_utils.cxx and simple `Logger` class.